components/product-carousel/product-carousel.makeswift.ts

```diff
@@ -21,6 +21,7 @@ const fetchCategories = async (): Promise<Category[]> => {
 
 export const props = {
   title: TextInput({ label: 'Title', defaultValue: 'Products', selectAll: true }),
+  className: Style(),
```

Comment:
I would use this guy:

```
className: Style({ properties: [Style.Margin, Style.Padding, Style.Width] }),
```

By default `Style` comes with a bunch of things like border radius that aren't relavent to this component.